### ISSUE TITLE
Cannot load checkpoint for pretraining

### ISSUE THREAD

**Issue by dangne** [yoheikikuta/bert-japanese#24]
Hi!

I am trying to fine-tuning the predicting masked token task on my corpus by running the run_pretraining.py but I cannot load your model checkpoint.

This is what I did
```
%%time
!python3 '/content/bert-japanese/src/run_pretraining.py' \
    --input_file='/content/gdrive/My Drive/Japanese OCR/train.tfrecord' \
    --output_dir='/content/gdrive/My Drive/Japanese OCR/bert-wiki-ja' \
    --init_checkpoint='/content/gdrive/My Drive/Japanese OCR/bert-wiki-ja/model.ckpt-1400000' \
    --use_tpu=True \
    --tpu_name='grpc://10.50.229.186:8470' \
    --do_train=True \
    --do_eval=False \
    --max_eval_steps=50 \
```
And I got this results
```
Unsuccessful TensorSliceReader constructor: Failed to get matching files on /content/gdrive/My Drive/Japanese OCR/bert-wiki-ja/model.ckpt-1400000: Unimplemented: File system scheme '[local]' not implemented (file: '/content/gdrive/My Drive/Japanese OCR/bert-wiki-ja/model.ckpt-1400000')
	 [[node checkpoint_initializer_204 (defined at tensorflow-1.15.2/python3.6/tensorflow_core/python/framework/ops.py:1748) ]]
```

**Comment by yoheikikuta** [comment]
You set `--init_checkpoint` as a local file system path which cannot be accessible from the TPU server.
Please see https://cloud.google.com/tpu/docs/troubleshooting?hl=en#cannot_use_local_filesystem.
